readme.md

```diff
@@ -70,10 +78,11 @@ To test run warp on all example contracts:
 bin/warp test
 ```
 
-For this to work, you must be able to run
+For this to work, you must have the cairo-lang package installed.
```

Comment:
We should just add the `pip install -r requirments.txt` to the instructions for testing